apps/sms/views/conversation/js/compose.js

```diff
@@ -4,10 +4,7 @@
 /*global Settings, Utils, Attachment, MozActivity, SMIL,
```

Comment:
> Yeah, in the Java world you would take an interface as parameter so you wouldn't have access to the full object (unless you'd cast but this is bad practice). 

Yep, in C# world too, but we're not there unfortunately :)

> I think Compose.setupLock would work fine in a splitted view app, but I don't like it in the main app architecture as this is really a changing state;

Yeah, if we didn't want to split views eventually, we wouldn't have problem that this PR is going to solve :)